i18n_scripts/combine_translations_files.py

```diff
@@ -0,0 +1,105 @@
+import os, localizable
+from collections import OrderedDict
+
+
+def translation_file_of(module):
+    translation_file = os.path.join(modules_dir, module, module, 'en.lproj', 'Localizable.strings')
+    return translation_file
+
+def get_modules_to_translate(modules_dir):
+    """
+    Retrieve the names of modules that have translation files for a specified language.
+
+    Parameters:
+        modules_dir (str): The path to the directory containing all the modules.
+
+    Returns:
+        list of str: A list of module names that have translation files for the specified language.
+    """
+    dirs = [directory for directory in os.listdir(modules_dir) if
+            os.path.isdir(os.path.join(modules_dir, directory))]
+
+    modules_list = []
+    for module in dirs:
+        translation_file = translation_file_of(module)
+        if os.path.exists(translation_file) and os.path.isfile(translation_file):
```

Comment:
this should be enough:

```suggestion
        if os.path.isfile(translation_file):
```

Reply:
done